jaeger_client/tracer.py

```diff
@@ -37,9 +37,11 @@
 from .span_context import SpanContext
 from .metrics import Metrics, LegacyMetricsFactory, MetricsFactory
 from .utils import local_ip
-from .sampler import Sampler
-from .reporter import BaseReporter
-from .throttler import Throttler
+
+if TYPE_CHECKING:
```

Comment:
What was wrong explicit imports?

Reply:
It can lead to import cycles. But let me remove it